Project Contribution Choice.md

```diff
@@ -0,0 +1,14 @@
+<h1 align="center"> Potential projects to contribute to</h1>
```

Comment:
pick one and move to the appropriate language folder, e.g. `projects/python/numpy.md`